### PR TITLE
add unit test for YamlOrchestrationConfiguration

### PR DESCRIPTION
**Pull request by llk984145406**
ISSUSE_ID.
For #4439
Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-api/src/test/java/org/apache/shardingsphere/orchestration/center/yaml/config/YamlOrchestrationConfigurationTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public final class YamlOrchestrationConfigurationTest {
+    
+    @Test
+    public void assertInstanceConfigurationMap() {
```
Comment (dongzl) [inline sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-api/src/test/java/org/apache/shardingsphere/orchestration/center/yaml/config/YamlOrchestrationConfigurationTest.java]: I suggest this method‘s name is `assertInstanceConfigurationMapWithConstructMethod`.

File: sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-api/src/test/java/org/apache/shardingsphere/orchestration/center/yaml/config/YamlOrchestrationConfigurationTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        Map<String, YamlInstanceConfiguration> instanceConfigurationMap = new HashMap<>();
+        YamlOrchestrationConfiguration yamlOrchestrationConfiguration = new YamlOrchestrationConfiguration(instanceConfigurationMap);
+        assertThat(yamlOrchestrationConfiguration.getInstanceConfigurationMap(), is(instanceConfigurationMap));
+    }
+    
+    @Test
+    public void assertSetInstanceConfigurationMap() {
```
Comment (dongzl) [inline sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-api/src/test/java/org/apache/shardingsphere/orchestration/center/yaml/config/YamlOrchestrationConfigurationTest.java]: I suggest this method‘s name is `assertInstanceConfigurationMapWithSetMethod`.

Comment (menghaoranss) [reply sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-api/src/test/java/org/apache/shardingsphere/orchestration/center/yaml/config/YamlOrchestrationConfigurationTest.java]: I agree with these suggestion.

**Pull request merged**